### ISSUE TITLE
Uncaught TypeError: Cannot set property '0' of undefined

### ISSUE THREAD

**Issue by martip-vccp** [DrewDahlman/Mason#50]
I'm getting this error right out of the box, testing it with the examples, using latest version from bower, any idea why?

THanks

**Comment by DrewDahlman** [comment]
Hmm not sure - just put this together http://jsfiddle.net/bvmp3gqe/ using fluid example

**Comment by martip-vccp** [comment]
Could it be related that my wrapper div has a fixed height?

**Comment by DrewDahlman** [comment]
Yep. That would cause it. It needs a flex height in order to size properly.

**Comment by martip-vccp** [comment]
Hi @DrewDahlman , I'm trying to do a horizontal grid, so I need a fixed height, is there any way of doing that? How would I need to modify the library to achieve it?

Thanks!

**Comment by tamtrong** [comment]
Hi @DrewDahlman , your mansonjs is cool. However, when I create 2 grids in the same page ( #grid1 & #grid2), the Error "Cannot set property '0' of undefined" always appears despite I config the grids seperately:
- $("#grid1").mason(...)
- $("#grid2").mason(...)
Could you please give me some ideas about this? 

Thanks!

**Comment by bl0wfish** [comment]
Applying `position:relative` to the container fixes it for me.
The problem is caused by the calculation of a block's top offset. If positioning is not relative it will be the offset to the top of the page, instead of the container:
https://github.com/DrewDahlman/Mason/blob/758d4ed3a45e0b8a6ab8a55f923dd465386bc722/src/mason.coffee#L301
